diff -rupN coursebuilder/app.yaml cb2/app.yaml
--- coursebuilder/app.yaml	2014-08-28 16:12:59.250529516 -0700
+++ cb2/app.yaml	2014-08-28 16:13:41.755146713 -0700
@@ -18,6 +18,7 @@ env_variables:
     modules.data_source_providers.data_source_providers
     modules.dashboard.dashboard
     modules.i18n_dashboard.i18n_dashboard=disabled
+    modules.lti.lti
     modules.manual_progress.manual_progress
     modules.mapreduce.mapreduce_module
     modules.notifications.notifications
diff -rupN coursebuilder/scripts/run_all_tests.py cb2/scripts/run_all_tests.py
--- coursebuilder/scripts/run_all_tests.py	2014-08-28 16:12:59.310530387 -0700
+++ cb2/scripts/run_all_tests.py	2014-08-28 16:13:56.727364054 -0700
@@ -31,6 +31,17 @@ import threading
 
 # all test classes with a total count of tests in each
 ALL_TEST_CLASSES = {
+    'tests.ext.lti.functional_tests.FieldsTest': 10,
+    'tests.ext.lti.functional_tests.LaunchHandlerTest': 8,
+    'tests.ext.lti.functional_tests.LoginHandlerTest': 7,
+    'tests.ext.lti.functional_tests.LTIToolTagTest': 3,
+    'tests.ext.lti.functional_tests.RedirectHandlerTest': 2,
+    'tests.ext.lti.functional_tests.RuntimeTest': 7,
+    'tests.ext.lti.functional_tests.SecurityParserTest': 9,
+    'tests.ext.lti.functional_tests.SerializerTest': 2,
+    'tests.ext.lti.functional_tests.ToolsParserTest': 7,
+    'tests.ext.lti.functional_tests.UrljoinTest': 6,
+    'tests.ext.lti.functional_tests.ValidationHandlerTest': 21,
     'tests.functional.admin_settings.AdminSettingsTests': 2,
     'tests.functional.admin_settings.HtmlHookTest': 11,
     'tests.functional.admin_settings.JinjaContextTest': 2,
